### ISSUE TITLE
RetroBar universally reduces framerate in Windows with certain games running in the background

### ISSUE THREAD

**Issue by Crunch64** [dremin/RetroBar#909]
(Windows 10)
Whenever I tab out of some games that are set to a lower framerate then what my monitor allows (i.e 60 vs 165), then the rest of my open applications and desktop will also use that framerate, making everything feel sluggish. Exiting RetroBar fixes this instantly.

This behavior doesn't happen with every game, so I'm unsure why some do this and some don't. My reported refresh rate in settings remains unchanged.

**Comment by dremin** [comment]
That’s pretty strange! Does the problem persist if you enable the software rendering option in RetroBar properties?

**Comment by Crunch64** [comment]
Yes, same problem persists. Difficult to demonstrate by recording it on my end unfortunately, but the difference does feel noticeable.

**Comment by Mertsudo** [comment]
this issue still persists please fix
